projects/angular/src/forms/styles/_checkbox.clarity.scss

```diff
@@ -88,15 +96,13 @@
       display: inline-block;
     }
 
-    $indetermSassBrdrColorVar: forms-variables.$clr-forms-checkbox-indeterminate-border-color;
-    $indetermCssBrdrColorVar: clr-forms-checkbox-indeterminate-border-color;
     input[type='checkbox'].clr-indeterminate + .clr-control-label::before,
     input[type='checkbox']:indeterminate + .clr-control-label::before {
       border: variables.$clr-global-borderwidth solid;
       @include mixins.css-var(
         border-color,
-        $indetermCssBrdrColorVar,
-        $indetermSassBrdrColorVar,
+        clr-forms-checkbox-border-color,
+        forms-variables.$clr-forms-checkbox-indeterminate-border-color,
         variables.$clr-use-custom-properties
       );
```

Comment:
Looks like both of these are `indeterminate` so they should use the same variable. `clr-forms-checkbox-border-color` is not needed here.

Reply:
corrected in commit b3df0d8